content/english/articles/showcases/tx-indexer/index.md

```diff
@@ -0,0 +1,140 @@
+
+---
+
+## What Does tx-indexer Do?
+
+At a high level:
+
+- **Indexes chain data**: It connects to your Gno chain via a specified `--remote` node URL and listens for new blocks and transactions.  
+- **Stores data locally**: It uses PebbleDB to store indexed data on disk.  
+- **Serves that data**: It provides a JSON-RPC 2.0 service (both HTTP and WebSocket) and a GraphQL endpoint to query or subscribe to new events.  
+
+For example, if you’re curious about how many contracts have been deployed on your chain, or how many transactions a certain address has recently made, the **tx-indexer** can help you gather that data. 
+
+### GraphQL!? That Sounds Interesting!
+
+While **tx-indexer** also provides standard JSON-RPC endpoints, the GraphQL interface is particularly convenient for modern web and analytics workflows. You can craft complex queries that filter by message type, block height, or transaction content and get exactly the data you need.
+
+Furthermore, the GraphQL endpoint provides **subscriptions**, which allow you to receive real-time events. This is ideal for dashboards that update with fresh data every few seconds.
+
+---
+
+## Getting Started
+
+### 1. Build and Run
+
+First, clone the repository and build the binary:
+
+```shell
+git clone https://github.com/gnolang/tx-indexer.git
+cd tx-indexer
+make build
+```
+
+Then, run the **tx-indexer** and point it to your Gno chain:
+
+```shell
+./build/tx-indexer start --remote https://rpc.gno.land --db-path indexer-db
+```
+
+This will index data from `https://rpc.test4.gno.land` and store it in `indexer-db`. The GraphQL endpoint (and JSON-RPC endpoint) will be available by default on port `8546` (configurable via the `--listen-address` flag).
+
+> **Note**  
+> The WebSocket endpoint is always at `ws://<listen-address>/ws`. You’ll need the WS port if you plan on receiving subscription events over WebSocket.
+
+### 2. Access the GraphQL Playground
+
+Once the indexer is running, you can open your browser to:
+
+```
+http://localhost:8546/graphql
+```
+
+This launches the GraphQL Playground—an interactive interface where you can run queries, view schema documentation, and experiment with filters.
+
+---
+
+## GraphQL in Action
+
+### Real-Time Subscriptions with GraphQL
+
+An extensive review of the GraphQL queries that are possible with the tx-indexer is beyond the scope of this article. Future documentation delving into this in more detail may eventually be found in the gno.land documentation or tutorials, but as a simple example, consider an event-driven workflow such as updating a dashboard or sending alerts when new blocks appear. the GraphQL subscription feature can make this simple.
+
+```graphql
+subscription {
+  blocks(filter: {}) {
+    height
+    version
+    chain_id
+    time
+    proposer_address_raw
+  }
+}
+```
+
+When you run the above subscription in the Playground, you’ll see real-time updates with each new block. You can imagine hooking this into:
```

Comment:
Just to be safe, we should reference the playground as GraphQL Playground, to avoid confusion with the gno playground